.github/infrastructure/terraform/conformance/pubsub/aws/snssqs/snssqs.tf

```diff
@@ -0,0 +1,94 @@
+resource "aws_sns_topic" "testTopic" {
+  name = "testTopic"
+  tags = {
+    dapr-topic-name = "testTopic"
+  }
+}
+
+resource "aws_sns_topic" "multiTopic1" {
+  name = "multiTopic1"
+  tags = {
+    dapr-topic-name = "multiTopic1"
+  }
+}
+
+resource "aws_sns_topic" "multiTopic2" {
+  name = "multiTopic2"
+  tags = {
+    dapr-topic-name = "multiTopic2"
+  }
+}
+
+resource "aws_sqs_queue" "testQueue" {
```

Comment:
Just cross-checking: if we also need to configure a dlq for any of the test

Reply:
YES! Good catch!

Reply:
The conformance tests do not require a dlq. The terraform file for the certification tests should have one however.